File: list/common_items.py
# Write a Python program to find common items from two lists.
# input
# color1 = "Red", "Green", "Orange", "White"
# color2 = "Black", "Green", "White", "Pink"
# output
# {'Green', 'White'}


def common_items():
    color1 = get_colors(1)
    color2 = get_colors(2)

    print(color1 & color2)

def get_colors(primordiality):
    n = int(input("Say how many colors you set in your color{}: ".format(primordiality)))
    colors = []

    for i in range(1, n+1):
        color = input("{}.-Color: ".format(i))
        colors.append(color)

    return set(colors)

common_items()

File: 3_list/common_items.rb
# Write a Python program to find common items from two lists.
# input
# color1 = "Red", "Green", "Orange", "White"
# color2 = "Black", "Green", "White", "Pink"
# output
# {'Green', 'White'}


def commom_items
  color1 = colors 1
  color2 = colors 2

  print color1 & color2
  puts
end



def colors(primordiality)
  print "How many colors your set #{primordiality} has?: "

  colors_count = gets.chomp.to_i
  colors = []

  colors_count.times do |index|
    print "#{index}.-Color: "
    color = gets.chomp

    colors << color
  end

  colors  
end


commom_items
